tests/test_ops/test_tensorrt_preprocess.py

```diff
@@ -41,7 +41,6 @@ def export_nms_module_to_onnx(module, onnx_file):
         output_names=['output'])
 
     onnx_model = onnx.load(onnx_file)
-    onnx.checker.check_model(onnx_model)
```

Comment:
Maybe it is not necessary to remove this checker. Could skip torch==1.3.1 in the unittest.

Reply:
Added version check before calling `onnx.checker.check_model`.

Or did you mean changes to `build.yml`?
Example:
```
build_cpu:
    ...
      - name: Run unittests and generate coverage report (torch == 1.3.1)
        if: ${{ matrix.torch == 1.3.1 }}
        run: |
          pip install -r requirements/test.txt
          coverage run --branch --source=mmcv -m pytest tests/ --ignore test_ops/test_tensorrt_preprocess.py
          coverage xml
          coverage report -m
      - name: Run unittests and generate coverage report (torch > 1.3.1)
        if: ${{ matrix.torch > 1.3.1 }}
        run: |
          pip install -r requirements/test.txt
          coverage run --branch --source=mmcv -m pytest tests/
          coverage xml
          coverage report -m
```